### PR TITLE
Posts can have new lines

### PR DESCRIPTION
**Pull request by carlfjones**
Posts can have newlines in the text area and once they are posted

### CODE REVIEW

File: spec/features/posts_can_have_a_newline_spec.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+require 'date'
+
+RSpec.feature "Format", type: :feature do
+  scenario "User can use newlines in posts" do
+    visit "/posts"
+    click_link "New post"
+    fill_in "Message", with: "Hello, world!\nHello back at ya!"
+    click_button "Submit"
+    expect(page).to have_content("Hello, world!\nHello back at ya! - #{Date.today.to_s}")
```
Comment (ajosephides) [inline spec/features/posts_can_have_a_newline_spec.rb]: Just a question - when I run on a server I see that the date appears on its own line and not encased in any <p>
One we may need to revisit when it comes to styling later.

File: app/views/posts/new.html.erb [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <%= form_for @post do |form| %>
```
Comment (ajosephides) [inline app/views/posts/new.html.erb]: Any reason form_with isn't used here?

Comment (l-palermo) [reply app/views/posts/new.html.erb]: form_with join the two previous form_for and form_tag and is a new feature of rails 5.1.
If we want to make our code uniform we need to decide whether to change all the form_form with form_with.

**Comment by l-palermo** [comment]
Thanks Carl.

**Pull request merged**